loader/loader.c

```diff
@@ -4693,7 +4693,7 @@ VkResult loader_create_instance_chain(const VkInstanceCreateInfo *pCreateInfo, c
         inst->instance = *created_instance;
     }
 
-    if (pCreateInfo->enabledLayerCount > 0 && pCreateInfo->ppEnabledLayerNames != NULL) {
+    if (res == VK_SUCCESS && pCreateInfo->enabledLayerCount > 0 && pCreateInfo->ppEnabledLayerNames != NULL) {
```

Comment:
Nit but this if statement could be folded into the if statement above. IE move the `}` on line 4696 down to 4707, then change `if(res == VK_SUCCESS && ...` to just `if(...`

Reply:
If you prefer it that way, sure I can change it. I was thinking about the same thing, but wanted to keep the change minimal for review purposes. I'll update the PR.

Reply:
Yeah, I can see how the original change is minimal. But, seeing how the code is structured, combining the if's together reads better.